client/post-editor/controller.js

```diff
@@ -177,6 +175,15 @@ module.exports = {
 		renderEditor( context, postType );
 	},
 
+	exitPost: function( context, next ) {
+		const postId = getPostID( context );
+		const siteId = getSelectedSiteId( context.store.getState() );
```

Comment:
@youknowriad Following the logic from boot, I don't think we render anything until the initial state has finished loading. It may have something to do with it being a `page.exit` handler, which we don't use in very many (any?) other parts of Calypso.